### ISSUE TITLE
Mouse input not working?

### ISSUE THREAD

**Issue by yamamushi** [twitchintheshell/twitch-master#23]
Are we launching qemu with the wrong flags or something? The stream voted on mouse movement commands and they didn't seem to do anything.

Also, we should probably remove the mouse x/y position output in chat, and maybe put it in the viewer?

I think Z needs to reset to 0.

I also think that x/y need to be saved. When something like this happens:

 mouse_move_dx 100

Qemu is getting this:

 mouse_move 100 0 

And the previous y coordinates aren't being passed in, so the mouse would always snap to 0?

**Comment by niemal** [comment]
This is exactly how I was told it works by the qemu developer channel. If it doesn't work then there are these possibilities:
- We are missing some flags when executing qemu.
- I don't know.

x and y are saved already btw....

**Comment by niemal** [comment]
Actually no, even if an axis (e.g. x or y) would be 0 and the other wouldn't, the mouse would move. The only thing that makes sense is that the point (0, 0) is at the center of the screen. So if I change x to something other than 0, the mouse would move up or down.

Something is wrong with the qemu and not with the actual code.

**Comment by yamamushi** [comment]
Yeah for sure, I tried inputing mouse movement manually but it wouldn't take :-/
